srcpkgs/cozy/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'cozy'
 pkgname=cozy
-version=1.0.1
+version=1.0.2
 revision=1
 build_style=meson
 hostmakedepends="pkg-config glib-devel python3 python3-distro python3-peewee
  python3-mutagen python3-gobject-devel gettext desktop-file-utils"
-makedepends="glib-devel python3-distro python3-peewee python3-mutagen
- python3-gobject-devel libhandy1-devel"
-depends="python3 python3-peewee gst-libav gst-plugins-good1 gst1-python3
- python3-mutagen python3-distro python3-apsw python3-packaging python3-pytz
- python3-requests python3-gobject libhandy1"
+makedepends="glib-devel granite-devel python3-distro python3-peewee
+ python3-mutagen python3-gobject-devel libhandy1-devel"
+depends="granite python3 python3-peewee gst-libav gst-plugins-good1
```

Comment:
Can you remove `python3` from here? And ideally make the changes be in a single line each, the diff ended up bigger than it had to be, I think.